packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -67,8 +64,8 @@ export default Vue.extend({
 		CommunityPackageCard,
 	},
 	async mounted() {
-		await this.$store.dispatch('communityNodes/fetchInstalledPackages');
 		try {
+			await this.$store.dispatch('communityNodes/fetchInstalledPackages');
```

Comment:
this should be in its own try/catch with error notification... it's important to show errors here because otherwise we would be showing the wrong state..
the second dispatch for the count is not important so it does not need error notification